src/View/Helper/TimeHelper.php

```diff
@@ -86,35 +86,35 @@ public function nice($dateString = null, $timezone = null, $locale = null)
     }
 
     /**
-     * Returns true if given datetime string is today.
+     * Returns true, if the given datetime string is today.
```

Comment:
Because thats the English rule.
A comma is only placed on those conditional sentences if the "if" comes first:

    If I do this, I will ...